descriptors/ModuleDescriptor-template.json

```diff
@@ -510,7 +516,9 @@
       { "name": "DB_DATABASE", "value": "okapi_modules" },
       { "name": "DB_QUERYTIMEOUT", "value": "60000" },
       { "name": "DB_CHARSET", "value": "UTF-8" },
-      { "name": "DB_MAXPOOLSIZE", "value": "5" }
+      { "name": "DB_MAXPOOLSIZE", "value": "5" },
+      { "name": "KAFKA_HOST", "value": "10.0.2.15"},
+      { "name": "KAFKA_PORT", "value": "9092" }
```

Comment:
Introducing Kafka as a new dependency to the module means that this version of the module cannot be used with the same infrastructure dependencies as previous versions. 

I think that means the major version of the module should be changed (and news / release notes provided) to communicate that change.

Reply:
Updated NEWS.md